kubernetes/nodecollection.go

```diff
@@ -402,6 +407,7 @@ func testNodeConn(config KubeAgentConfig, client *http.Client, mask EndpointMask
 		if err != nil {
 			return false, err
 		}
+		log.Info("Using /stats/container endpoint")
```

Comment:
So you want to log "Unable to reach /stats/container endpoint"?

Reply:
There are a few ways to approach it, but we probably want to check either the value of `cs` or the availability of `NodeContainerEndpoint` in the mask to determine what to log. It might be nice to log both available and not available states during the beta period.